tools/packchk/include/CreateModel.h

```diff
@@ -26,11 +27,15 @@ class CreateModel
   bool CheckForOtherPdscFiles(const std::string& pdscFullPath);
   bool AddPdsc(const std::string& pdscFile, bool bSkipCheckForOtherPdsc = false);
   bool AddRefPdsc(const std::set<std::string>& pdscRefFiles);
+  bool SetPackXsd(const std::string& packXsdFile);
   bool ReadAllPdsc();
   bool PrintPdscFiles(std::list<std::string>& pdscFiles);
 
 private:
   RteModelReader m_reader;
+  XmlValidator m_xmlValidator;
```

Comment:
Making this a dynamically allocated pointer would solve two problems:
1) no need to include XmlValidator.h in the header, forward declaration is sufficient.
2) the object can be created only if needed (validation is enabled)